### DISCUSSION TITLE
Support how to test app-service !

### DISCUSSION THREAD

**Discussion by MILANOFCOFEE** [edgexfoundry/community#99] [Q&A]
Hello everyone, I got a problem when testing app-service. I built an app-service custom. This app-service with behavior GET all core-data (local) then POST to Cloud. So can I test the behavior of this app-service? Please give me advice, thank you so much!

**Comment by herrdragon** [comment]
What exactly is the part you need to test? If the part of the cloud receiving your data, there are many free endpoints testing sites. I use https://public.requestbin.com/, one url supports all http verbs. If the get data from core-data, run your app in debug mode to see what is getting. Hope this helps you.

**Reply by MILANOFCOFEE** [reply]
Thank you so much!
